mcp_server/server.py

```
@@ -0,0 +1,966 @@
+            else None
+        ),
+        "current_metrics": stats.get("current_window", {}),
+        "thresholds": stats.get("thresholds", {}),
+    }
+
+
+@mcp.tool()
+@handle_mcp_error
+async def detect_anomalies(
+    sensitivity: float = 2.0,
+    window_minutes: Optional[int] = None,
```

Ruff (0.14.10)

Unused function argument: `window_minutes`

(ARG001)